packages/hooks/use-archive-infinite-scroll/types.ts

```diff
@@ -0,0 +1,86 @@
+import { IntersectionOptions } from "../use-infinite-scroll/types";
+
+/**
+ * Options received by the {@link useArchiveInfiniteScroll} hook.
+ */
+export type UseArchiveInfiniteScrollOptions = {
+  /**
+   * The number of pages that the hook should load automatically before
+   * switching to manual fetching.
+   */
+  limit?: number;
+
+  /**
+   * A boolean indicating if this hook should be active or not. It can be
+   * useful in situations where users want to share the same component for
+   * different types of Archives, but avoid doing infinite scroll in some of
+   * them.
+   *
+   * @defaultValue true
+   */
+  active?: boolean;
+
+  /**
+   * The intersection observer options for fetching.
+   */
+  fetchInViewOptions?: IntersectionOptions;
+
+  /**
+   * The intersection observer options for routing.
+   */
+  routeInViewOptions?: IntersectionOptions;
+};
+
+/**
+ * Object returned by the {@link useArchiveInfiniteScroll} hook.
+ */
+export type UseArchiveInfiniteScrollOutput = {
+  /**
+   * An array of the existing pages. Users should iterate over this array in
+   * their own layout.
+   */
+  pages: {
+    /**
+     * A unique key to be used in the iteration.
+     */
+    key: string;
+
+    /**
+     * The link of this page.
+     */
+    link: string;
+
+    /**
+     * If this page is the last page. Useful to add separators between pages,
+     * but avoid adding it for the last one.
+     */
+    isLast: boolean;
+
+    /**
+     * The Wrapper component that should wrap the real `Archive` component.
+     */
+    Wrapper: React.FC;
```

Comment:
Same here.

Reply:
I answered in the previous comment: https://github.com/frontity/frontity/pull/652#discussion_r583545884